frontend-shared/styles/mixins/utils.scss

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
I suppose sticking transitions in a `utils` SASS module for now is OK...we'll see how this module grows.